CoreJs-part2.js

```diff
@@ -0,0 +1,83 @@
+//CoreJS. Part 2
+//1. Write a JavaScript function that takes an array of numbers as input and returns the sum of all even numbers in the array.
+function sumOfEven(values) {
+  let sum = 0;
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] % 2 == 0) {
+      sum += values[i];
+    }
+  }
+  return sum;
+}
+console.log(
+  "Task1. sum of even values in array = " +
+    sumOfEven([1, 2, 4, 6, 7, 1, 7, 7, 2])
+);
+
+//2. Write a JavaScript function that takes two input values and returns the concatenation of the two values as a string.
+
+function valuesSumAsString(a, b) {
+  return "" + a + b;
+}
+
+console.log("Task2. sum of values 4 and 5 = " + valuesSumAsString(4, 5));
+console.log(
+  "Task2. sum of values Hello and 5 = " + valuesSumAsString("Hello", 5)
+);
+console.log("Task2. sum of values 4 and 5 = " + valuesSumAsString(4, null));
+console.log(
+  "Task2. sum of values 4 and 5 = " + valuesSumAsString(undefined, 5)
+);
+
+//3. Write a JavaScript function that checks if a given number is positive, negative, or zero and returns the corresponding string: "Positive", "Negative", or "Zero".
+function checkNumber(a) {
+  return a > 0
+    ? "Positive"
+    : a < 0
+    ? "Negative"
+    : a == 0
```

Comment:
Change it to triple equals `a === 0`.

Try to run `checkNumber('')`. If you use `==` empty string `''` will be coerced to number.

![image](https://github.com/girlinitworld/CoreJS/assets/118353174/46fe87d1-24f1-4754-93e9-b69dbd085dee)


https://262.ecma-international.org/13.0/#sec-islooselyequal